website/source/docs/commands/connect/envoy.html.md.erb

```diff
@@ -39,7 +39,7 @@ proxy configuration needed.
  - `-grpc-addr=<addr>` - Address of the Consul agent with `grpc` port. This can
    be an IP address or DNS address, but it must include the port. This can also
    be specified via the CONSUL_GRPC_ADDR environment variable. In Consul 1.3 and
-   later, the default value is http://127.0.0.1:8502, and https can optionally
+   later, the default value is 127.0.0.1:8502, and https can optionally
```

Comment:
@i0rek I don't exactly understand what the original author meant by `https can optionally be used instead`.

Reply:
We support http and https endpoints:

https://github.com/hashicorp/consul/blob/58c0c101afd06470bf9d096838ac262751ece2b7/command/connect/envoy/envoy.go#L199-L211

Does that help?

Reply:
Yes. I've updated the documentation to make it a bit clearer that https can be used by passing the value as `https://127.0.0.1:8502`

Reply:
Sorry, reverted it back to the first commit because adding passing `grpc-addr` with `https://` prefix doesn't seem to work in my setup. I'll let this PR just be about correcting the default value of `grpc-addr`.